Rakefile

```diff
@@ -55,3 +55,24 @@ end
 task :test do
   ruby("test/run-test.rb")
 end
+
+namespace :doc do
+  task :add_permalink do
+    news_md = File.read("doc/text/news.md")
+    base_url = "https://github.com/test-unit/test-unit/pull/"
+
+    applied_typed_permalink = news_md.gsub(/(?<pre>\[GitHub#(?<ref>\d+)\])(?<post> ?\[(?<type>Report|Patch))/) do
+      if Regexp.last_match[:type] == "Report"
+        base_url = "https://github.com/test-unit/test-unit/issues/"
```

Comment:
If we find any `Report` type, all rest `[GitHub#XXX]`s use `issues/` not `pull/`.

BTW, GitHub accepts `issues/XXX` for pull requests. So we can always use `issues/`.

Reply:
>If we find any Report type, all rest [GitHub#XXX]s use issues/ not pull/.

Sorry for the bug 🙇‍♂️  Fixed in https://github.com/test-unit/test-unit/pull/198/commits/03fbf37aa5c7710a917f7a569eeb911b96f38430

>So we can always use issues/.

Updated with `issues` suffix in https://github.com/test-unit/test-unit/pull/198/commits/1f3ab61ad38cfd906b6ebe47d459b12edbac2429.